src/profile/voyages/tag/vsca.js

```diff
@@ -80,7 +87,124 @@ try{
         if (u.data.url.indexOf("http") !== 0 && u.data.url.indexOf("/") !== 0) {
           u.data.url = location.protocol + "//" + u.data.url;
         }
+        //START Customised VSCA tag call
+	window.vsca_pageTag = window.vsca_pageTag || {};
+        function widgetCall() {
+	  vsca_pageTag.config={};
+	  vsca_pageTag.config.siteId="agency-funnel";
+	  vsca_pageTag.config.vsca_version="2.1";
+	  vsca_pageTag.config.country="FR";
+	  vsca_pageTag.config.language="fr";
+          
+	  vsca_pageTag.contextData={};
+          vsca_pageTag.contextData.products=[];
+          if(b.utagPageName=="page.car.search.list" || b.utagPageName=="page.car.search.list.aws"){ 
+	    vsca_pageTag.config.pageId="CarSearch";	    
+	    vsca_pageTag.contextData.pageContext="searchResult";
+            var product0 = {
+                "productType":"Car",
+                "outwardDepartureDate":b.checkOutDate,
```

Comment:
Yes We could do that, But I wrote the script like this to keep it clear and simple to debug. 
For instance, From CarSearch page to Infosiet, I should redefine 80% of Product context, so I preferred separate on two different  statement.
Anyway, No problem to make this improvement form our side.